operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -90,47 +122,55 @@ A DNS SRV record is structured as follows:
 - Name: The domain name for which this record is valid.
 - Other elements were omitted for clarity and brevity.
 
+```text
+<service>.<protocol>.<domain name>
+```
+
 For example, DNS SRV record might look like this:
 
 ```text
-_sbps-hsn._tcp.my-system.my-site-domain.net
+_sbps-hsn._tcp.my-system.my-site-domain
 ```
 
 The following is an explanation of the values used in this example:
 
 - `_sbps-hsn` is the symbolic name of the service.
+  - In this example, it is using the High Speed Network (hsn).
 - `_tcp` is the transport protocol.
-- `my-system.my-site-domain.net` is the domain name for which the record is valid. The domain name includes the system name `my-system` and the site domain name `my-site-domain.net`.
+- `my-system.my-site-domain` is the domain name for which the record is valid.
+  - The domain name includes the system name `my-system` and the site domain name `my-site-domain`.
+
+To use the Node Management Network (NMN) for content projection, the service is set to `_sbps-nmn`.
+  - `"rootfs_provider_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-nmn._tcp.my-system.my-site-domain:300"`
+  
+To use the High Speed Network (HSN) for content projection, the service is set to `_sbps-hsn`.
+  - `"rootfs_provider_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-hsn._tcp.my-system.my-site-domain:300"`
+
+**Reminder:** The DNS SRV record should contain the system's _actual_ DNS domain.
+In this example, the `system-name` is `my-system` and the site domain name is `my-site-domain`. These need to be
+replaced with the system's _actual_ system name and site DNS Domain. Refer to the [options above](#discovering-my-system-and-my-site-domain) to determine the system-name and site-domain.
+
 
 ### `<client discovery timeout in seconds>`
 
 The timeout, in seconds, for attempting to mount the `netroot` via SBPS.
```

Comment:
This is a query rather than a comment. What is 'netroot' ?

Reply:
This should have been rootfs.